Documentation/Usage.md

```diff
@@ -143,84 +143,86 @@ To `import Clibgit`, the package manager requires that the libgit2 library has
 been installed by a system packager (eg. `apt`, `brew`, `yum`, etc.). The
 following files from the libgit2 system-package are of interest:
 
-    /usr/local/lib/libgit2.dylib      # .so on Linux
-    /usr/local/include/git2.h
+    /opt/homebrew/lib/libgit2.dylib      # .so on Linux
+    /opt/homebrew/include/git2.h
```

Comment:
I would probably leave this as `/usr/local` since not everyone is using homebrew, and change the note to say it may be in `/opt/homebrew` etc

Reply:
Cleaned this up in this and other spots.